crates/control/src/dribble_path_plan.rs

```diff
@@ -0,0 +1,98 @@
+use color_eyre::Result;
+use context_attribute::context;
+use framework::{AdditionalOutput, MainOutput};
+use nalgebra::Point2;
+use spl_network_messages::Team;
+use std::f32::consts::PI;
+use types::{
+    configuration::Behavior, FieldDimensions, GameControllerState, PathObstacle, PathSegment,
+    WorldState,
+};
+
+use crate::behavior::walk_to_pose::WalkPathPlanner;
+
+#[context]
+
+pub struct CreationContext {}
+
+#[context]
+pub struct CycleContext {
+    pub world_state: Input<WorldState, "world_state">,
+    pub field_dimensions: Parameter<FieldDimensions, "field_dimensions">,
+    pub configuration: Parameter<Behavior, "behavior">,
```

Comment:
Please sort the members within a group alphabetically

Reply:
done